scripts/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+cleanup() {
+  code=$?
+  echo 'Something went wrong! Reinstalling dependencies...'
+  yarn >/dev/null
+  exit $code
+}
+trap cleanup ERR
+
+echo 'Installing dependencies...'
+yarn >/dev/null
+
+echo 'Bundling source code...'
+webpack >/dev/null
+
+if [ -n "$1" ]
+then
+  echo 'Bumping version...'
+  node scripts/bump-version.js $1
+fi
+
+echo 'Removing node modules...'
+rm -rf node_modules/
+
+echo 'Zipping files...'
+zip -qr package.zip .
```

Comment:
slightly dangerous if there's anything in the current dir, like private keys, etc.

better to create a zipfile from an export: http://stackoverflow.com/a/163769